src/schedule.rs

```diff
@@ -316,6 +322,25 @@ impl ScheduleFields {
     }
 }
 
+/// Wrapper over [`Schedule`] used if equivalence of its [`TimeUnitSpec`]s needs to be tested
+/// # Example
+/// ```
+/// use cron::{Schedule, TimeUnitSpecEq};
+/// use std::str::FromStr;
+/// let schedule_1 = Schedule::from_str("@weekly").unwrap();
+/// let schedule_2 = Schedule::from_str("0 0 0 * * 1 *").unwrap();
+/// assert_ne!(schedule_1, schedule_2);
+/// assert_eq!(TimeUnitSpecEq(schedule_1), TimeUnitSpecEq(schedule_2));
+/// ```
+#[derive(Clone, Debug, Eq)]
+pub struct TimeUnitSpecEq(pub Schedule);
+
+impl PartialEq for TimeUnitSpecEq {
+    fn eq(&self, other: &TimeUnitSpecEq) -> bool{
+        self.0.fields == other.0.fields
```

Comment:
Rather than vending a new type that achieves this, how would you feel about having something like:

```rust
impl Schedule {
  // ... 
  pub fn ordinals_eq(other: &Schedule) -> bool {
    self.fields == other.fields
  }
  // ...
}
```
and allowing users to define their own wrapper type that implements `PartialEq` in terms of `ordinals_eq`? That alleviates the pressure to provide a type with a perfect name.

Reply:
Sure no problem

Reply:
Nitpick: Maybe we should call it `timeunitspec_eq`? Because Ordinals is not exposed?

Reply:
> Nitpick: Maybe we should call it timeunitspec_eq? Because Ordinals is not exposed?

Oh, good point! I wish I liked the type name `TimeUnitSpec` more, but we'll still have time to revise that before 1.0 if I can think of something better. Let's go with `timeunitspec_eq`.